pulp_container/app/viewsets.py

```diff
@@ -359,6 +359,46 @@ def copy_manifests(self, request, pk):
         )
         return OperationPostponedResponse(result, request)
 
+    @swagger_auto_schema(
+        operation_description="Trigger an asynchronous task to build an OCI image from a "
+                              "Dockerfile. A new repository version is created with the new image "
+                              "and tag. This API is tech preview in Pulp Container 1.1. Backwards "
+                              "compatibility when upgrading is not guaranteed.",
+        operation_summary="Build a Dockerfile",
```

Comment:
Build from a Dockerfile

Reply:
I am going to rename this to "Build from a Containerfile". The Buildah docs say that the two files have the exact same syntax. I'll explain this in the docs. What do you think?

Reply:
sounds good to me!